operator/mapper/redis.go

```diff
@@ -0,0 +1,47 @@
+package mapper
```

Comment:
Warning, personal opinion ahead:

I don't like the `type PostgreSQLMapper struct{}` and associated method receivers pattern. It strikes me as a way to use classes in a language without classes. 

I also don't see why mappers specific to certain controllers should be in a shared package. They don't interest outside of that specific package.

I made a compromise here: I didn't use the empty struct pattern but moved the map functions into the `mapper` package, even though I'd really prefer them to be in the `rediscontroller` package.

Reply:
> well, the associated method receivers pattern is actually the closest thing to a class. Look at interface implementations for example. Whether they're called structs or classes makes little difference for a Dev, so I'm fine with that.

mh yeah, that's where we disagree I guess ;)

>  I don't like static functions that have repeatedly the same name over and over again in the function names just to differentiate between other, similar functions.

I believe that shows a structural problem with the current code. The `mappers` package is fine to have, but the `postgres` (or redis for that matter) specific code does IMO not belong there. It's of no use to other packages since it's highly specific to the controller in question. 

>  At that time I was expecting that most types or aliases can be shared between different DB services, so I prepared the mapper package as a separate translation layer between Exoscale API and our provider API.

Ok that's what I thought. 
I think it's fine to have it, but the things we can share only. I.e. postgres stuff should not belong there IMO. I'll be moving the functions for redis back and we can refactor postgres in a separate PR if necessary.

Reply:
> If you think that doesn't really apply, I'm fine with moving these mappings to their DB service-specific packages and that would also not "require" empty structs due to the naming problem.

Actually that would solve the naming problem if we move the mapper inside it's own package. We cannot reuse the mappers due to the oapi design and that's a pity because a lot of the db services are having the same attributes.

Reply:
yeah, let's move the DB specific mappers to the controller packages :+1:

Reply:
will do in a separate PR for postgres :+1: